client/blocks/comment-detail/style.scss

```diff
@@ -0,0 +1,261 @@
+%comment-detail__ellipsis {
+	overflow: hidden;
+	text-overflow: ellipsis;
+	white-space: nowrap;
+}
+
+%comment-detail__gray-link {
+	color: $gray;
+	cursor: pointer;
+	fill: $gray;
+	&:focus,
+	&:hover {
+		color: $blue-wordpress;
+		fill: $blue-wordpress;
+	}
+}
+
+.comment-detail.card {
+	font-size: 14px;
+	margin: 0 auto;
+	padding: 0;
+	transition: margin .15s linear;
+
+	&.is-expanded {
+		margin: 16px auto;
+	}
+	&:not( .is-expanded ):hover {
+		box-shadow: 0 0 0 1px $gray, 0 2px 4px lighten( $gray, 20% );
+		z-index: 1;
+	}
+
+	.gridicon {
+		margin-right: 4px;
+		vertical-align: middle;
+	}
+}
+
+.comment-detail__author-avatar {
+	flex-shrink: 0;
+	img {
+		border-radius: 50%;
+	}
+}
+.comment-detail__author-info {
+	@extend %comment-detail__ellipsis;
```

Comment:
So if we find that we're repeating ourselves, we normally try to break things out as a components. Which would work well to change `%comment-detail__gray-link;` into a new comment link component or as a composed link inside of a comment compoennt.

`%comment-detail__ellipsis` is a bit tougher. A few options here:
1. keep the extends (though we'd need to convert to a mixin if we add media queries). See: https://github.com/Automattic/wp-calypso/blob/master/assets/stylesheets/shared/mixins/_mixins.scss#L17-L31
2. just repeat the rules in the related classes